### ISSUE TITLE
Where is the postcard that X sent to Y in chap1 problem?

### ISSUE THREAD

**Issue by jaeyoo** [JackHidary/quantumcomputingbook#13]

**Comment by rmlarose** [comment]
It must be still in transit :) Thanks for catching this, we will update the problem set asap.

**Comment by jaeyoo** [comment]
Thank you @rmlarose. plus, I want to ask a tiny question. In chapter02.pdf, there is a sentence like `would lead to operating effectively on 4-vectors as C^2 is equivalent to R^2`. I think it meant `C^2 is equivalent to R^4`, right?

**Comment by rmlarose** [comment]
Yes that's right @jaeyoo I will fix this typo in chp 2 -- thanks!

We'll also get the missing figure uploaded to the chp 1 problem set soon.

**Comment by jaeyoo** [comment]
@rmlarose Thank you for clarification 👍
